samples/client/petstore/java/rest-assured-jackson/src/main/java/org/openapitools/client/model/ModelApiResponse.java

```diff
@@ -35,7 +35,7 @@
   ModelApiResponse.JSON_PROPERTY_TYPE,
   ModelApiResponse.JSON_PROPERTY_MESSAGE
 })
-@JsonTypeName("ApiResponse")
+@JsonTypeName("_ApiResponse")
```

Comment:
Is this change intentional? Change in the JsonTypeName will be a breaking change without fallback. This PR won't get far as is.

Reply:
Hmm, no, certainly not. I'll look into it.

Reply:
I fixed this by modifying `AbstractJavaCodegen.fromName` so that it keeps the original name here, rather than calling `escapeReservedWord` for these. I tried changing it directly in `DefaultCodegen`, but that caused other problems as usual (in the Kotlin generator, this time). Please take another look.